app/services/websocket.py

```
@@ -22,13 +30,104 @@ async def connect(self, websocket: WebSocket, user_id: str):
+
+    async def subscribe_to_market_data(self, websocket: WebSocket, symbols: List[str]):
+        """Subscribe WebSocket to market data updates for specific symbols."""
+        for symbol in symbols:
+            if symbol not in self.market_subscribers:
+                self.market_subscribers[symbol] = []
+            if websocket not in self.market_subscribers[symbol]:
+                self.market_subscribers[symbol].append(websocket)
+        
+        # Start market streaming if not already running
+        if not self.is_market_streaming:
+            asyncio.create_task(self._start_market_streaming())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)